src/repositories/taskInstanceRepository.js

```diff
@@ -0,0 +1,51 @@
+const sql = require('mssql');
+
+function taskInstanceRepository({ dataClient }) {
+  async function getTasks() {
+    const pool = await dataClient.getConnection();
+    const request = await pool.request();
+    request.output('response', sql.VarChar);
+
+    return new Promise((resolve, reject) => {
+      request.execute('SPGetTasks', (error, result) => {
+        if (error !== null && error !== undefined) {
+          reject(error);
+        } else {
+          const res = result.output.response == null ? '[]' : result.output.response;
```

Comment:
Use `===` and parse the out response here before returning it to the service / controller